sourcegraph/sourcegraph
# search: clean up feature flags

Remove experimental feature flags for notebooks, code monitoring, and search contexts as these features have been enabled by default for a long time now.

## Test plan

- Verify all three features still appear in the navbar and work as expected in other parts of the product (eg. search results action list).
- Verify all three features do NOT show when running `sg start oss`.
